src/client/app/containers/CompareChartContainer.js

```diff
@@ -26,6 +26,22 @@ function mapStateToProps(state) {
 	// Power used up to this point last week
 	let currentLastWeek = 0;
 	const soFar = moment().diff(moment().startOf('week'), 'days');
+	const delta = function (change) {
+		if (isNaN(change)) {
+			return '';
+		}
+		if (change < 0) {
+			return `You have used ${parseInt(change.toFixed(2).replace('.', ''), 10)}% less energy this week.`;
+		}
+		return `You have used ${parseInt(change.toFixed(2).replace('.', ''), 10)}% more energy this week.`;
```

Comment:
You should replace "You have used" with "{Meter name} has used {value}% more energy this week"